roles/common/tasks/proj_tests.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Verify Project exists - "{{ item }}"
+  ansible.builtin.shell: 
+    cmd: |
+      oc project "{{ item }}"
```

Comment:
This will switch to a different project. This may cause unexpected behaviour of subsequent commands are run without a `--namespace/-n` argument.
A better check would be

``oc project list | grep "{{ item}}"``

(please verify that this command is correct)

A more efficient approach might be to get the project list and then loop through the output to make sure that all the expected project are there.